packages/docs-browser/src/components/DocsView/DocsView.js

```diff
@@ -114,7 +130,11 @@ DocsView.propTypes = {
   match: PropTypes.object.isRequired,
   history: PropTypes.object.isRequired,
   navigationStrategy: PropTypes.object,
-  domainTypes: PropTypes.objectOf(PropTypes.string),
+  domainTypes: PropTypes.arrayOf(PropTypes.string),
+  rootNodes: PropTypes.arrayOf(PropTypes.shape({
+    model: PropTypes.string,
```

Comment:
https://tocco-docs.readthedocs.io/en/latest/framework/client/code-styleguide.html#tocco-variable-naming

here was my attempt to add consistency to the naming. so in this case it would rather be named "entityName".  :)